src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -178,6 +180,30 @@ public function getEnv($prefix, $name, \Closure $getEnv)
             return $env;
         }
 
+        if ('url' === $prefix) {
+            $parsedEnv = \parse_url($env);
+
+            if (false === $parsedEnv) {
+                throw new RuntimeException(sprintf('Invalid URL in env var "%s"', $name));
+            }
+
+            if ('/' === ($parsedEnv['path'][0] ?? '')) {
+                if (!isset($parsedEnv['scheme'], $parsedEnv['host'])) {
+                    throw new RuntimeException(sprintf('Invalid URL env var "%s": schema and host expected, %s given.', $name, $env));
+                }
+                // remove the '/' separator
+                $parsedEnv['path'] = substr($parsedEnv['path'], 1);
+            }
+
+            return $parsedEnv;
+        }
+
+        if ('query' === $prefix) {
```

Comment:
query_string?

Reply:
yep, you're right on that (too)!

fixed